src/com/expense/EqualExpense.java

```diff
@@ -0,0 +1,34 @@
+package com.expense;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EqualExpense extends Expense{
+
+	@Override
+	public List<ExpenseLog> splitExpense(String expenseDetails) {
+		String[] details = expenseDetails.split(" ");
```

Comment:
This should not be part of your design. Should have created the expense object and passed here from the main method itself. Parsing the input should not be done here.